package.json

```diff
@@ -28,6 +28,7 @@
     "url": "https://github.com/agenda/agenda/issues"
   },
   "dependencies": {
+    "debug": "^2.6.8",
```

Comment:
I don't believe so. You want people to be able to post logs if they have problems and they won't be able to turn it on on their server if it is in devDeps.

See in `express`: https://github.com/expressjs/express/blob/master/package.json#L36